### PR TITLE
New red slime speed potion

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Using water on a red slime extract will yield a speed potion.

Applying that speed potion to an item will paint it red and remove its slowdown.

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
https://www.google.co.uk/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0CAcQjRxqFQoTCPTCh4jK1cgCFcXrFAodLjAFIQ&url=http%3A%2F%2Fseajey.livejournal.com%2F156832.html&psig=AFQjCNGV_osj3eCLCOD5zDERFXTsjcWsIQ&ust=1445586108537253

**Comment by Dorsisdwarf** [comment]
fasta

**Comment by GunHog** [comment]
<b>DA RED WUNZ GO FASTA!!!</b>

**Comment by Aloraydrel** [comment]
WAAAAAAAAAAAAAAAAAGH

**Comment by Lumipharon** [comment]
Based as fuck.

**Comment by PKPenguin321** [comment]
im guessing this is for things like galoshes and clown shoes..?

**Comment by KorPhaeron** [comment]
@PKPenguin321 

And hardsuits, bombsuits, space suits, duffle bags,etc

**Comment by PKPenguin321** [comment]
oh, that's actually really cool

**Comment by KorPhaeron** [comment]
@Lumipharon

Not right now no. It probably wouldnt be that hard to do but it probably wouldnt be the most balanced thing in the world.

**Comment by Zilenan91** [comment]
best :+1:

**Comment by blobbernaut** [comment]
kor, it'd happen once every like 100 rounds that a mech is made and xenobiology gets done nice and proper and the red exctract gets into the hands of the mech. I don't see a problem with a MINOR buff.

**Comment by Zilenan91** [comment]
So how does this look on the hardsuits anyways?

**Comment by GunHog** [comment]
Here is how it looks with reactive tele armor and the RD's hardsuit.
screenshot 2015-10-24 14 49 43

**Comment by PKPenguin321** [comment]
if you apply it to a hardsuit, the color effect should automatically apply to the helmet, too
not 100% necessary but it would look nicer

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -297,7 +297,21 @@
 	M.mutator_used = TRUE
 	qdel(src)
 
+/obj/item/slimepotion/speed
+	name = "slime speed potion"
+	desc = "A potent chemical mix that will remove the slowdown from any item."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle3"
 
+/obj/item/slimepotion/speed/afterattack(obj/item/C, mob/user)
```
Comment (GunHog) [inline code/modules/research/xenobiology/xenobiology.dm]: Could this potentially runtime if used on things not in the ````obj/item```` path?

Comment (Razharas) [inline code/modules/research/xenobiology/xenobiology.dm]: It will

Comment (KorPhaeron) [inline code/modules/research/xenobiology/xenobiology.dm]: Added a check

**Comment by Razharas** [comment]
Sounds silly but lets find out

**Pull request merged**